benchmarks/api/fabric/workloads/compositeTx/read-write-assets.js

```diff
@@ -0,0 +1,129 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict';
+const bytes = (s) => {
+    return ~-encodeURI(s).split(/%..|./).length;
+};
+
+const writeOptions = {
+    read: 'true',
+    notRead: 'false',
+    random: 'random',
+  };
+
+const helper = require('../helper');
+const { WorkloadModuleBase } = require('@hyperledger/caliper-core');
+
+/**
+ * Workload module for the benchmark round.
+ */
+class ReadWriteAssetsWorkload extends WorkloadModuleBase {
+    /**
+     * Initializes the workload module instance.
+     */
+    constructor() {
+        super();
+        this.chaincodeID = '';
+        this.byteSize = 0;
+        this.readSize = 0;
+        this.writeSize = 0;
+        this.previouslyRead = writeOptions.random; //use enum to say the mode(include random)
```

Comment:
comment can be removed, variable name should be `writeMode` rather than previouslyRead